### ISSUE TITLE
Update parent version to 9.11

### ISSUE THREAD

**Issue by acotiuga** [xwikisas/application-filemanager#23]

**Comment by polx** [comment]
I've been investigating this a bit, so as to build myself a better debugging environment for another issue and met thus far the following issues which probably need a bit of discussion:

* the version 9 of XWiki introduces the use of the Maven Spoon Plugin with a [configuration](https://github.com/xwiki/xwiki-commons/blob/master/xwiki-commons-pom/pom.xml#L188) to prohibit File.deleteOnExit. While this is apparently a good intent, there are no replacement (as I could confirm in chat); also I do not see deleteOnExit to be fully correct as a restart may happen and the download be resumed. Wouldn't the correct deletion be after the download is finished (this needs to enrich the /temp/ action with a listener or a parameter) and some time has passed. The deletion should then also delete from the recent downloads. I also tried using finalize but that is prohibited by checkstyle. I am not sure where to start.

* Unit tests versions and probably dependencies need a bit of tuning. After resolving ambiguities due to the new version of Mockito, I encountered "A required class was missing" for any of the 3.0.0-Mx versions (x= 1, 3, or 4).
